projects/slipshow/programs/slipshow/tests/shell.nix

```diff
@@ -0,0 +1,38 @@
+{
+  sources,
+  ...
+}:
+{
+  name = "slipshow presentation";
+
+  nodes = {
+    machine =
+      { pkgs, ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs.slipshow
+          sources.examples.slipshow.demo-shell
+        ];
+
+        environment.etc."slipshow.md".source = pkgs.fetchurl {
+          url = "https://github.com/panglesd/slipshow/blob/v0.2.0/example/campus-du-libre/slipshow.md";
+          hash = "sha256-4+ow0GQ8RAIYDDM6rg69/X4aYs9GgqFSuEjTJmAoG8A=";
+        };
+
+      };
+  };
+
+  testScript =
+    { nodes, ... }:
+    ''
+      start_all()
+
+      # it may take around a minute to compile the file and serve it
+      machine.succeed("slipshow serve /etc/slipshow.md &")
```

Comment:
I think you have to `slipshow compile /etc/slipshow.md` first, or does it already do that?

This said, rendering doesn't look correct on my end either way:

![image](https://github.com/user-attachments/assets/9a7819f7-123c-4b5d-85fb-e40e4c2ac737)